src/UI/ManualImport/Movie/SelectMovieLayout.js

```diff
@@ -0,0 +1,101 @@
+var _ = require('underscore');
+var vent = require('vent');
+var Marionette = require('marionette');
+var Backgrid = require('backgrid');
+var MoviesCollection = require('../../Movies/MoviesCollection');
+var SelectRow = require('./SelectMovieRow');
+
+module.exports = Marionette.Layout.extend({
+    template  : 'ManualImport/Movie/SelectMovieLayoutTemplate',
+
+    regions : {
+        movie : '.x-movie'
+    },
+
+    ui : {
+        filter : '.x-filter'
+    },
+
+    columns : [
+        {
+            name      : 'title',
+            label     : 'Title',
+            cell      : 'String',
+            sortValue : 'sortTitle'
```

Comment:
Can you change that to title. sortTitle is currently undefined with most movies :/